zarr/tests/test_storage.py

```diff
@@ -900,6 +900,16 @@ def test_context_manager(self):
 except ImportError:  # pragma: no cover
     sqlite3 = None
 
+try:
+    import pymongo
+except ImportError:  # pragma: no cover
+    pymongo = None
+
+try:
+    import redis
+except ImportError:  # pragma: no cover
+    redis = None
```

Comment:
Similar to comment above for mongo, what about adding a check here for a local server, as a mechanism for skipping redis tests if server not available?

Reply:
done.